app/controllers/concerns/accessible.rb

```diff
@@ -9,6 +9,9 @@ def execute_if_accessible(object, redirect: true)
       flash[:error] = 'only owner can do that'
       redirect_to root_path if redirect
     end
+
+  rescue ActiveRecord::RecordInvalid
```

Comment:
Generally, rescuing ActiveRecord built-in errors in some custom catch blocks is a code smell. Especially if this `rescue` block is in a aside module, not in a class itself: the surface for unexpected behavior is tremendously high.

Can you at least add a verbose comment in your code, with a specific flow when this `rescue` saves your day?